angular/package.json

```diff
@@ -1,6 +1,8 @@
 {
   "name": "fleetui",
   "version": "0.0.0",
+  "description": "Fleet UI",
+  "repository": "https://github.com/xuwang/fleet-ui.git",
```

Comment:
Thank you for contributing to this project. :)
Would you change the repository url to original? (https://github.com/purpleworks/fleet-ui)